packages/pds/tests/seeds/client.ts

```diff
@@ -197,7 +206,71 @@ export class SeedClient {
     return repost
   }
 
-  userRef(did: string): UserRef {
+  async createScene(creator: string, handle: string) {
+    const res = await this.client.app.bsky.actor.createScene(
+      {
+        handle,
+      },
+      {
+        headers: this.getHeaders(creator),
+        encoding: 'application/json',
```

Comment:
Is it possible we can nix this `encoding`?  I think it's intended to be the default when there's an input object.

Reply:
merp https://github.com/bluesky-social/atproto/pull/304#discussion_r1015839741